### DISCUSSION TITLE
Multitenancy with Multi DB

### DISCUSSION THREAD

**Discussion by foxpcteam** [refinedev/refine#1689] [Q&A] [answered]
**Hi Guys, First I want to say thank you for this great project.**

As for my questions.

You wrote a tutorial for "Multitenancy" with a single db, which is not "really Multitenancy" that use for each tenancy for each "client".
Can you make simple example as for real word (SaaS) that can use refine with Multitenancy that create for each "client" his own tenancy(db).

One more question, what the best way to give a "client" API access to his tenancy (data) only ( Is there something built in refine that can help with this?).

**By the way, your demo for Multitenancy tutorial with Appwrite on Codesandbox not working (can't login because of error)**

**Comment by omeraplak** [comment, marked as answer]
Hey @foxpcteam ,
Thank you for contacting us. The Multi-tenancy example we created show you how to manage with Refine (frontend). Our goal in this guide was not to set an example for the Backend.
